releasenotes/notes/35883.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+releaseNotes:
+- |
+  **Enabled** discovering cross-cluster, same network, endpoints for headless services. This can be disabled by setting
```

Comment:
nit: may mention this is tied to the (off by default) DNS settings.

Reply:
right, thanks!